apps/dotcom/client/src/tla/components/TlaFileError/TlaFileError.tsx

```diff
@@ -35,15 +35,15 @@ export function TlaFileError({ error }: { error: unknown }) {
 		case TLSyncErrorCloseEventReason.NOT_AUTHENTICATED: {
 			return (
 				<TlaFileErrorContent
-					header="Private file"
-					para1="Contact the owner to request access."
+					header="Sign in"
```

Comment:
We're using `Log in` and `Sign in` inconsistently, would be good to pick one and stick with it.